feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt

```diff
@@ -277,6 +280,15 @@ class SessionDetailFragment : DaggerFragment() {
         }, builderAction)
     }
 
+    private fun computeLineCount(fullText: String): Int {
```

Comment:
We should use `TextUtils#ellipsize` or `measureText` to detect this.

`TextUtils#ellipsize` can ellipsize the given text actually so it's really reliable.
`getTextBounds` rounds values but `measureText` can treat values as float, so measureText has higher accuracy.

Reply:
I prefer TextUtils#ellipsize because of https://github.com/DroidKaigi/conference-app-2019/pull/595/files#diff-a45d9aed66a757a483bc9cdf2b196c9aR251

Reply:
Wow, I don't know such methods, thank you! 👍 
I'll improve it 💪